35-Seminar-hee2/35-Seminar-hee2/Source/Toss/View/AppScoreView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  AppScoreView.swift
+//  35-Seminar-hee2
+//
+//  Created by 김희은 on 10/25/24.
+//
+
+import SnapKit
+import UIKit
+import Then
+
+final class AppScoreView: UIView {
+    
+    private let titleLabel = UILabel().then {
+        $0.text = "평가 및 리뷰"
+        $0.font = .systemFont(ofSize: 24, weight: .bold)
+        $0.textColor = .label
+        $0.textAlignment = .left
+    }
+    private let scoreLabel = UILabel().then {
+        $0.text = "4.4"
+        $0.font = .systemFont(ofSize: 40, weight: .heavy)
+        $0.textColor = .label
+        $0.textAlignment = .center
+    }
+    private let baseScoreLabel = UILabel().then {
+        $0.text = "5점 만점"
+        $0.font = .systemFont(ofSize: 16)
+        $0.textColor = .secondaryLabel
+        $0.textAlignment = .center
+    }
+    public lazy var allReviewButton = UIButton().then {
+        $0.setTitle("모두 보기", for: .normal)
+        $0.backgroundColor = .systemBackground
+        $0.setTitleColor(.tintColor, for: .normal)
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setUI()
+        setLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setUI() {
+        [titleLabel, scoreLabel, baseScoreLabel, baseScoreLabel, allReviewButton].forEach {
+            self.addSubview($0)
+        }
+    }
+    
+    private func setLayout() {
+        titleLabel.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(20)
+            $0.leading.equalToSuperview().offset(20)
```

Comment:
superview를 기준으로 오토레이아웃을 잡을 때에는 inset을 사용해주시는 것이 일반적이랍니닷

추가로 top과 leading의 inset값이 같이 때문ㄴ에 아래와 같이 바꿔주는 것도 좋겠네욤!
```suggestion
            $0.top.leading.equalToSuperview().inset(20)
```

Reply:
우왓 몰랏습니다 감사해용-!